src/Serializer/AbstractItemNormalizer.php

```diff
@@ -89,12 +92,25 @@ public function supportsNormalization(mixed $data, string $format = null, array
 
     /**
      * {@inheritdoc}
+     * @deprecated since Symfony 6.3, use "getSupportedTypes(?string $format)" instead
      */
     public function hasCacheableSupportsMethod(): bool
     {
+        trigger_deprecation('symfony/serializer', '6.3', 'The "%s()" method is deprecated, use "getSupportedTypes()" instead.', __METHOD__);
+
         return true;
     }
 
+    public function getSupportedTypes(?string $format)
+    {
+        return [
+            HalItemNormalizer::class => true,
+            JsonApiItemNormalizer::class => true,
+            JsonLdItemNormalizer::class => true,
+            ItemNormalizer::class => true,
+        ];
```

Comment:
I think this must return `['object' => false]`.

The description of the `getSupportedTypes` method indicates `a result cannot be cached when it depends on the context or on the data.` which the `AbstractItemNormalizer` does.

Generally, this method should return which objects are supported by the normalizer and these listed classes are normalized on their own which, I think, will never be normalized itself.